Gemfile

```diff
@@ -60,6 +60,10 @@ gem "omniauth-rails_csrf_protection"
 # print different colors to the terminal
 gem 'colorize'
 
+# Bootstrap
+gem 'jquery-rails'
+gem 'bootstrap-sass'
```

Comment:
Why do you need this gem? We already had bootstrap installed the conventional way, and things are rendering very weird on a phone on this branch...

Reply:
Sorry, I did not see it. We can remove them from the Gemfile.